components/announcement_bar/version_bar/version_bar.tsx

```diff
@@ -59,7 +59,7 @@ export default class VersionBar extends React.PureComponent <Props, State> {
                                 id='version_bar.new'
                                 defaultMessage='A new version of Mattermost is available.'
                             />
-                            {' '}
+                            &nbsp;
```

Comment:
even when this works just fine, I don't see an extended use of the non breaking space across the app. So far what I have seen is either modify the text so it contains the space where necessary or the use of css margins to get the same result. My personal choice would be to add inline css to the anchor tag with half rem:
`                            <a
                                onClick={this.reloadPage}
                                style={{marginLeft: '.5rem'}}
                            >`